rpc/src/v1/traits/parity.rs

```diff
@@ -240,7 +241,7 @@ build_rpc_trait! {
 		/// Extracts Address and public key from signature using the r, s and v params. Equivalent to Solidity erecover
 		/// as well as checks the signature for chain replay protection
 		#[rpc(name = "parity_verifySignature")]
-		fn verify_signature(&self, bool, Bytes, H256, H256, U64) -> Result<RecoveredAccount>;
+		fn verify_signature(&self, bool, Bytes, H256, H256, u64) -> Result<RecoveredAccount>;
```

Comment:
I changed the type to `u64` because `ethereum_types::U64` can't be converted into a `u64` or some other type to perform arithmetics on

Reply:
This is going to change the signature of the method - we should be careful here. I would prefer to have all numbers (or at least the ones that can't be express in JS with enough precision) encoded as hex

Reply:
Wouldn't `low_u64()` or `as_u64()` work?

Reply:
Cool, both work just looked at the `From/Into` impls will change it back